back/flag-service/jest-integration.json

```diff
@@ -2,7 +2,7 @@
   "moduleFileExtensions": ["js", "json", "ts"],
   "rootDir": ".",
   "testEnvironment": "node",
-  "testRegex": ".spec-integration.ts$",
+  "testRegex": "\\.spec\\.ts$",
```

Comment:
This jest configuration shouldn't be modified to run all spec file. It is my bad.

The problem is that I created only one `test` script in `package.json`

We should have  `test` and `test:integration` that runs related tests.